tonic/src/transport/server/mod.rs

```diff
@@ -122,7 +120,7 @@ impl Default for Server<Identity> {
             tcp_keepalive: None,
             tcp_nodelay: false,
             http2_keepalive_interval: None,
-            http2_keepalive_timeout: None,
+            http2_keepalive_timeout: Duration::from_secs(20),
```

Comment:
I think this is fine can we add back the const just to make it easier to discover the value in the code?